applitools.config.js

```diff
@@ -0,0 +1,11 @@
+
+const applitoolsSettings = {
+  appName: 'Carbon Sage',
+  // Strict, Exact, Layout and Content - https://help.applitools.com/hc/en-us/articles/360007188591-Match-Levels
+  matchLevel: 'Strict',
+  envName: 'Development',
+  testName: 'my test',
```

Comment:
What are `testName` and `batchName`?

Reply:
The `testName` and `batchName` are the predefined values to use after in  `cypress/support/index.js` in line 99-100

Reply:
Deleted those line, tested without - works correctly, as we are overriding those values in `cypress/support/index.js`